### PR TITLE
feat: Greek Translations, Create el.json

### PR DESCRIPTION
**Pull request by sl45sms**
Greek Translation (el)

### CODE REVIEW

**Comment by WilcoFiers** [comment]
@sl45sms Wow thank you! Apologies for the delayed response. This one slipped through in the vacation. I'll need to find someone who can help us review this, which from past experience can take a while. So heads up on that.

**Comment by WilcoFiers** [comment]
Just another quick ping to let you know we haven't forgotten. As I warned this can be slow, but I'm still working on it!

**Comment by sl45sms** [comment]
> Just another quick ping to let you know we haven't forgotten. As I warned this can be slow, but I'm still working on it!

Great, not rush, take the time as needed, thanks a lot.

**Review by straker — CHANGES_REQUESTED** [review]
Apologies for the long delay in getting this reviewed. We're hoping to pull it into the 4.8 release happening soon. Thanks again for taking the time to translate.

Below are a few questions and suggestions. Importantly some of the wording has changed since you made this translation and I wanted to point that out to see if you wanted to update the wording to reflect the current values.

File: locales/el.json [hunk 1]
```diff
@@ -0,0 +1,1067 @@
...
+      "aria-allowed-attr": {
+        "description": "Διασφαλίζει ότι τα χαρακτηριστικά ARIA επιτρέπονται για το ρόλο του στοιχείου",
+        "help": "Τα στοιχεία πρέπει να χρησιμοποιούν μόνο επιτρεπόμενα χαρακτηριστικά ARIA"
+      },
+      "aria-allowed-role": {
+        "description": "Διασφαλίζει ότι το χαρακτηριστικό role έχει την κατάλληλη τιμή για το στοιχείο",
+        "help": "Ο ρόλος ARIA θα πρέπει να είναι κατάλληλος για το στοιχείο"
+      },
+      "aria-command-name": {
+        "description": "Διασφαλίζει ότι κάθε ARIA σε κουμπί,σύνδεσμο ή στοιχείο μενού έχει ένα προσβάσιμο όνομα",
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json:21]: it is clear for the Greek programmers, but for compatibility reasons I will add them.

File: locales/el.json [hunk 2]
```diff
@@ -0,0 +1,1067 @@
...
+        "description": "Διασφαλίζει ότι τα κουμπιά έχουν ευδιάκριτο κείμενο",
+        "help": "Τα κουμπιά πρέπει να έχουν ευδιάκριτο κείμενο"
+      },
+      "bypass": {
+        "description": "Διασφαλίζει ότι κάθε σελίδα έχει τουλάχιστον έναν μηχανισμό για τον χρήστη να παρακάμπτει την πλοήγηση και να μεταβαίνει απευθείας στο περιεχόμενο",
+        "help": "Η σελίδα πρέπει να έχει τρόπο για την παράκαμψη επαναλαμβανόμενων μπλοκ"
+      },
+      "color-contrast-enhanced": {
+        "description": "Διασφαλίζει ότι η αντίθεση μεταξύ των χρωμάτων προσκηνίου και φόντου πληροί τα όρια αναλογίας αντίθεσης WCAG 2 AAA",
+        "help": "Τα στοιχεία πρέπει να έχουν επαρκή χρωματική αντίθεση"
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: i will do that.

File: locales/el.json [hunk 3]
```diff
@@ -0,0 +1,1067 @@
...
+          "hidden": "Η τιμή του aria-errormessage `${data.values}` δεν μπορεί να αναφέρει ένα κρυφό στοιχείο"
+        },
+        "incomplete": {
+          "singular": "βεβαιωθείτε ότι η τιμή του aria-errormessage `${data.values}` αναφέρεται σε ένα υπάρχον στοιχείο",
+          "plural": "βεβαιωθείτε ότι οι τιμές του aria-errormessage `${data.values}` αναφέρονται σε υπάρχοντα στοιχεία",
+          "idrefs": "δεν είναι δυνατός ο προσδιορισμός του εάν το στοιχείο aria-errormessage υπάρχει στη σελίδα: ${data.values}"
+        }
+      },
+      "aria-hidden-body": {
+        "pass": "Δεν υπάρχει χαρακτηριστικό arria-hidden στο σώμα του εγγράφου",
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: fixed

File: locales/el.json [hunk 4]
```diff
@@ -0,0 +1,1067 @@
...
+      },
+      "valid-scrollable-semantics": {
+        "pass": "Το στοιχείο έχει έγκυρη σημασιολογία για ένα στοιχείο στη σειρά εστίασης.",
+        "fail": "Το στοιχείο έχει μη έγκυρη σημασιολογία για ένα στοιχείο στη σειρά εστίασης."
+      },
+      "color-contrast-enhanced": {
+        "pass": "Το στοιχείο έχει επαρκή χρωματική αντίθεση ${data.contrastRatio}",
+        "fail": {
+          "default": "Το στοιχείο έχει ανεπαρκή χρωματική αντίθεση ${data.contrastRatio} (χρώμα προσκηνίου: ${data.fgColor}, χρώμα φόντου: ${data.bgColor}, μέγεθος γραμματοσειράς: ${data.fontSize}, βάρος γραμματοσειράς: ${data.fontWeight}). Αναμενόμενος λόγος αντίθεσης ${data.expectedContrastRatio}",
+          "fgOnShadowColor": "Το στοιχείο έχει ανεπαρκή χρωματική αντίθεση ${data.contrastRatio} μεταξύ του χρώματος του προσκηνίου και του χρώματος σκιάς (χρώμα προσκηνίου: ${data.fgColor}, χρώμα σκιάς κειμένου: ${data.shadowColor}, μέγεθος γραμματοσειράς: ${data.fontSize} , βάρος γραμματοσειράς: ${data.fontWeight}). Αναμενόμενος λόγος αντίθεσης ${data.expectedContrastRatio}",
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: fixed

File: locales/el.json [hunk 5]
```diff
@@ -0,0 +1,1067 @@
...
+        }
+      },
+      "color-contrast": {
+        "pass": {
+          "default": "Το στοιχείο έχει επαρκή χρωματική αντίθεση ${data.contrastRatio}",
+          "hidden": "Το στοιχείο είναι κρυμμένο"
+        },
+        "fail": {
+          "default": "Το στοιχείο έχει ανεπαρκή χρωματική αντίθεση ${data.contrastRatio} (χρώμα προσκηνίου: ${data.fgColor}, χρώμα φόντου: ${data.bgColor}, μέγεθος γραμματοσειράς: ${data.fontSize}, βάρος γραμματοσειράς: ${data.fontWeight}). Αναμενόμενος λόγος αντίθεσης ${data.expectedContrastRatio}",
+          "fgOnShadowColor": "Το στοιχείο έχει ανεπαρκή χρωματική αντίθεση ${data.contrastRatio} μεταξύ του χρώματος προσκηνίου και σκιάς (χρώμα προσκηνίου: ${data.fgColor}, χρώμα σκιάς κειμένου: ${data.shadowColor}, μέγεθος γραμματοσειράς: ${data.fontSize} , βάρος γραμματοσειράς: ${data.fontWeight}). Αναμενόμενος λόγος αντίθεσης ${data.expectedContrastRatio}",
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: fixed

File: locales/el.json [hunk 6]
```diff
@@ -0,0 +1,1067 @@
...
+      },
+      "meta-viewport-large": {
+        "pass": "Η ετικέτα <meta> δεν εμποδίζει τη σημαντική μεγέθυνση σε κινητές συσκευές",
+        "fail": "Η ετικέτα <meta> περιορίζει τη μεγέθυνση σε κινητές συσκευές"
+      },
+      "meta-viewport": {
+        "pass": "Η ετικέτα <meta> δεν απενεργοποιεί το ζουμ σε κινητές συσκευές",
+        "fail": "Η ετικέτα ${data} στην ετικέτα <meta> απενεργοποιεί το ζουμ σε κινητές συσκευές"
+      },
+      "target-offset": {
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json:823]: changed

File: locales/el.json [hunk 7]
```diff
@@ -0,0 +1,1067 @@
...
+        },
+        "incomplete": {
+          "default": "Το στοιχείο με αρνητικό tabindex έχει ανεπαρκές μέγεθος (${data.width}px επί ${data.height}px, θα πρέπει να είναι τουλάχιστον ${data.minSize}px επί ${data.minSize}px). Είναι στόχος;",
+          "contentOverflow": "Το μέγεθος του στοιχείου δεν ήταν δυνατό να προσδιοριστεί με ακρίβεια λόγω του περιεχομένου υπερχείλισης",
+          "partiallyObscured": "Το στοιχείο με αρνητικό tabindex δεν έχει επαρκές μέγεθος επειδή είναι μερικώς κρυμμένο (το μικρότερο διάστημα είναι ${data.width}px επί ${data.height}px, θα πρέπει να είναι τουλάχιστον ${data.minSize}px επί ${data.minSize} px). Είναι στόχος;",
+          "partiallyObscuredNonTabbable": "Ο στόχος έχει ανεπαρκές μέγεθος επειδή είναι μερικώς καλυμμένος από έναν γείτονα με αρνητικό tabindex (το μικρότερο διάστημα είναι ${data.width}px επί ${data.height}px, θα πρέπει να είναι τουλάχιστον ${data.minSize}px επί ${data .minSize}px). Είναι στόχος ο γείτονας;"
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: fixed

File: locales/el.json [hunk 8]
```diff
@@ -0,0 +1,1067 @@
...
+      "accesskeys": {
+        "description": "Διασφαλίζει ότι η τιμή κάθε χαρακτηριστικού accesskey είναι μοναδική",
+        "help": "η τιμή κάθε χαρακτηριστικού accesskey πρέπει να είναι μοναδική"
+      },
+      "area-alt": {
+        "description": "Διασφαλίζει ότι τα στοιχεία <area> των χαρτών εικόνας έχουν εναλλακτικό κείμενο",
+        "help": "Τα ενεργά στοιχεία <area> πρέπει να έχουν εναλλακτικό κείμενο"
+      },
+      "aria-allowed-attr": {
+        "description": "Διασφαλίζει ότι τα χαρακτηριστικά ARIA επιτρέπονται για το ρόλο του στοιχείου",
```
[the remark this answers is not in the document]
Comment (sl45sms) [reply locales/el.json]: changed

**Comment by straker** [comment]
@sl45sms Am I correct to assume that this PR is ready for review again with all your updates?

**Comment by sl45sms** [comment]
> @sl45sms Am I correct to assume that this PR is ready for review again with all your updates?

Yes please, review the updates. Thanks.

File: locales/el.json (hunk 8 above)
[the remark this answers is not in the document]
Comment (ibrahimosman1127) [reply locales/el.json]: Thanks

**Review by straker — APPROVED** [review]

**Comment by straker** [comment]
Thanks for the translations! Approved for security

**Pull request merged**